geomesa-accumulo/geomesa-accumulo-tools/src/main/resources/org/locationtech/geomesa/accumulo/tools/ingest-libjars.list

```diff
@@ -15,6 +15,9 @@ gson
 gt
 htrace
 joda
+json4s-ast
```

Comment:
this file is slowly getting out of control. As in maybe we should have a folder called lib/mapreduce